--- /usr/share/phpPgAdmin/conf/config.inc.php.org	2014-01-14 17:07:15.000000000 +0900
+++ /usr/share/phpPgAdmin/conf/config.inc.php	2017-08-05 14:31:30.045251006 +0900
@@ -90,7 +90,7 @@
 	// will be denied. Only set this false once you have read the FAQ and
 	// understand how to change PostgreSQL's pg_hba.conf to enable
 	// passworded local connections.
-	$conf['extra_login_security'] = true;
+	$conf['extra_login_security'] = false;
 
 	// Only show owned databases?
 	// Note: This will simply hide other databases in the list - this does
