--- /etc/nginx/conf.d/default.conf.org	2017-06-03 16:59:34.160891933 +0900
+++ /etc/nginx/conf.d/default.conf	2017-06-03 16:37:11.000000000 +0900
@@ -5,10 +5,8 @@ server {
     #charset koi8-r;
     #access_log  /var/log/nginx/log/host.access.log  main;
 
-    location / {
-        root   /usr/share/nginx/html;
-        index  index.html index.htm;
-    }
+    root   /usr/share/nginx/html;
+    index  index.html index.htm index.php;
 
     #error_page  404              /404.html;
 
@@ -25,15 +23,14 @@ server {
     #    proxy_pass   http://127.0.0.1;
     #}
 
-    # pass the PHP scripts to FastCGI server listening on 127.0.0.1:9000
+    # pass the PHP scripts to FastCGI server listening on unix:/var/run/php-fpm/php-fpm.sock
     #
-    #location ~ \.php$ {
-    #    root           html;
-    #    fastcgi_pass   127.0.0.1:9000;
-    #    fastcgi_index  index.php;
-    #    fastcgi_param  SCRIPT_FILENAME  /scripts$fastcgi_script_name;
-    #    include        fastcgi_params;
-    #}
+    location ~ \.php$ {
+        fastcgi_pass   unix:/var/run/php-fpm/php-fpm.sock;
+        fastcgi_index  index.php;
+        fastcgi_param  SCRIPT_FILENAME  $document_root$fastcgi_script_name;
+        include        fastcgi_params;
+    }
 
     # deny access to .htaccess files, if Apache's document root
     # concurs with nginx's one
